packages/mux-uploader/src/mux-uploader.ts

```diff
@@ -233,16 +211,9 @@ class MuxUploaderElement extends globalThis.HTMLElement implements MuxUploaderEl
     }
   }
 
-  get formatProgress(): (percent: number) => string {
-    return this._formatProgress ?? defaultFormatProgress;
-  }
-
-  set formatProgress(value: ((percent: number) => string) | null | undefined) {
-    this._formatProgress = value;
-  }
-
   setupRetry() {
     this.retryButton?.addEventListener('click', () => {
+      this.dispatchEvent(new CustomEvent('reset'));
```

Comment:
suggestion(non-blocking): Add this event to the event mappings types `MuxUploaderElementEventMap`. Can be a followup with other currently unmapped events (noticed some additional `// @ts-ignore` around events that should be resolved by this).

Reply:
Will address this in a followup ts cleanup